fe/fe-core/src/main/java/com/starrocks/connector/delta/DeltaUtils.java

```diff
@@ -113,4 +110,54 @@ public static RemoteFileInputFormat getRemoteFileFormat(String format) {
             throw new StarRocksConnectorException("Unexpected file format: " + format);
         }
     }
+
+    public static TPhysicalSchema getPhysicalSchema(StructType physicalSchema, Metadata metadata) {
+        String columnMappingMode = ColumnMapping.getColumnMappingMode(metadata.getConfiguration());
+        if (columnMappingMode.equals(ColumnMapping.COLUMN_MAPPING_MODE_ID) ||
+                columnMappingMode.equals(ColumnMapping.COLUMN_MAPPING_MODE_NAME)) {
+            TPhysicalSchema tPhysicalSchema = new TPhysicalSchema();
+            for (StructField field : physicalSchema.fields()) {
+                tPhysicalSchema.addToFields(getPhysicalSchemaField(field, columnMappingMode));
+            }
+            return tPhysicalSchema;
```

Comment:
Is there a case that meets the following conditions?
1. column mapping mode is id or name
2. the TPhysicalSchemaField from getPhysicalSchemaField() has no physical name or field id